tests/unit/cluster/cluster-shards.tcl

```diff
@@ -0,0 +1,55 @@
+# Get the node info with the specific node_id from the
+# given reference node. Valid type options are "node" and "shard"
+proc get_node_info_from_shard {id reference {type node}} {
```

Comment:
move this to `tests/support/cluster_util.tcl`?

nit - I feel it is more natural to place `reference` in front of `id`, as the first argument. I also notice that this function is lifted from `cluster/tests/28-cluster-shards.tcl`. It can be confusing and error-prone to have two functions with the same name but different argument order so if you decide to take this suggestion we will need to update the other place as well.

```suggestion
proc get_node_info_from_shard {reference node_id {type node}} {
```

Reply:
Yeah, it's copy paste. Lot of usage is already there in 28-cluster-shards.tcl. Will avoid it, if you don't feel it's a blocker.

Reply:
Not a blocker to me